docs/data/data-grid/getting-started/getting-started.md

```diff
@@ -134,13 +117,12 @@ export default function App() {
 
 ## TypeScript
 
-In order to benefit from the [CSS overrides](/material-ui/customization/theme-components/#theme-style-overrides) and [default prop customization](/material-ui/customization/theme-components/#theme-default-props) with the theme, TypeScript users need to import the following types.
-Internally, it uses module augmentation to extend the default theme structure.
+To benefit from [CSS overrides](/material-ui/customization/theme-components/#theme-style-overrides) and [default prop customization](/material-ui/customization/theme-components/#theme-default-props) with the theme, TypeScript users must import the following types.
+These types use module augmentation to extend the default theme structure.
 
 ```tsx
+// Pro and Premium users: add `-pro` or `-premium` suffix to package name
```

Comment:
I'm still on the fence about which is preferable:

- list all three imports? (annoying for copy+paste)
- only include Community version? (could be unclear or just a minor annoyance for paid users)

Reply:
Not for this PR, but I was wondering if we could improve the overall experience of the Data Grid docs by having a plan switcher. Something like this, but it would apply to all of the snippets (and maybe demos) to only show the package that the user is interested in:

https://github.com/user-attachments/assets/a3910970-37fe-4b61-814a-5178e13d4b63

Reply:
Yeah that would be nice to have!